### PR TITLE
7584: Use pdf-lib for previewing PDF in modal

### PR DESCRIPTION
**Pull request by sutt0n**

### CODE REVIEW

File: web-client/src/applicationContext.js [hunk 1]
```diff
@@ -557,6 +557,7 @@ const applicationContext = {
     const pdfjsLib = await import('pdfjs-dist');
     const pdfjsWorker = await import('pdfjs-dist/build/pdf.worker.entry');
     pdfjsLib.GlobalWorkerOptions.workerSrc = pdfjsWorker;
+    console.log('pdfjsWorker', pdfjsLib.GlobalWorkerOptions.AnnotationFlag);
```
Comment (kkoskelin) [inline web-client/src/applicationContext.js]: Can this be removed?

File: web-client/src/views/ModalDialog.jsx [hunk 2]
```diff
@@ -102,6 +103,7 @@ export const ModalDialog = ({
             aria-modal="true"
             className={classNames('modal-dialog padding-205', className)}
             role="status"
+            style={{ maxHeight: '100vh' }}
```
Comment (kkoskelin) [inline web-client/src/views/ModalDialog.jsx]: This is now in the modal css and can be removed from here, right?

File: web-client/src/views/PDFPreviewModal.jsx [hunk 3]
```diff
@@ -22,23 +22,14 @@ export const PDFPreviewModal = connect(
   function PDFPreviewModal({
     cancelSequence,
     confirmSequence,
-    currentPage,
     loadPdfSequence,
-    pdfPreviewModal,
-    pdfPreviewModalHelper,
+    //pdfPreviewModal,
```
Comment (kkoskelin) [inline web-client/src/views/PDFPreviewModal.jsx]: is it okay to pull this out?

**Review by kkoskelin — CHANGES_REQUESTED** [review]
Just a little bit of housekeeping.  Nice work on the library-swap implementation.

**Review by kkoskelin — APPROVED** [review]

**Comment by matthopson** [comment]
Screen Shot 2021-01-22 at 10 57 36 AM
Tossing a mobile (hidden link) screenshot in here for good measure.

**Pull request merged**